githubAPI/octokit.js

```diff
@@ -0,0 +1,58 @@
+const { Octokit } = require("@octokit/rest");
+require('dotenv').config();
+
+const octokit = new Octokit({
+  auth: process.env.WEBHOOK_SECRET,
+  log: {
+    debug: () => {},
+    info: () => {},
+    warn: console.warn,
+    error: console.error
+  }
+});
+
+if(!octokit.auth) {
+  console.log('Missing authentication token. Please add correct environment variable to file .env!');
+  return undefined;
+}
+
+module.exports.asyncTryToken = async function() {
```

Comment:
I think no need to specify `async` in the function identifier? Also I think good where possible to follow the convention is beginning boolean methods/variables with `is`. E.g, 

```suggestion
const isValidToken = async () => {
```

Reply:
Added as part of single commit